roles/confluent.zookeeper/defaults/main.yml

```diff
@@ -22,6 +22,8 @@ zookeeper_service_environment_overrides:
   KAFKA_HEAP_OPTS: "-Xmx1g"
   KAFKA_OPTS: "{{ zookeeper_final_java_args | java_arg_build_out }}"
 
+# In zookeeper.properties, current node's hostname will be 0.0.0.0, meaning zookeeper listens on all interfaces
+zookeeper_current_node_hostname: 0.0.0.0
```

Comment:
i'm not sold we should default to 0.0.0.0 here because I am reading some zk security stuff, and in some cases its best to have a FDQN here. dont merge until we discuss

Reply:
plan is to default to inventory hostname, but add documentation